apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -336,13 +340,13 @@ WithTransforms.args = {
     visDomain: [-2, 1.5],
     isIndexAxis: true,
     showGrid: false,
-    flip: false,
+    flip: true,
   },
   ordinateConfig: {
     visDomain: [0, 2],
     isIndexAxis: true,
     showGrid: false,
-    flip: false,
+    flip: true,
```

Comment:
I now flip axes in the `WithTransform` story to be closer from the daiquiri-tomo set-up